oaknational/Oak-Web-Application
# PUPIL-641 - Feat/pupil 641/lesson listing page

## Description

Music year: 1972

create a boilerplate page which uses the lessonListingQuery

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2379--oak-web-application.netlify.thenational.academy/pupils/beta/programmes/biology-secondary-year-10-foundation-aqa/units/biological-molecules-and-enzymes/lessons
2. Click on each lesson
3. You should see it take you to correct url

### Acceptance criteria

- [x]  served at pupils/beta/programmes/PROGRAMME_SLUG/units/UNIT_SLUG/lessons
- [x]  display unit name , subject and year
- [x]  list lessons as links
- [x]  clicking on lesson navigates to lesson via browse path
